test/jdk/javax/swing/JComponent/7154030/bug7154030.java

```diff
@@ -50,7 +52,7 @@
  * @library /lib/client/
  * @build Util
  * @build ExtendedRobot
- * @run main bug7154030
+ * @run main/othervm -Dsun.java2d.uiScale=1 bug7154030
```

Comment:
The "sun.java2d.uiScale" does not affect the sizes of the frame on macOS, Is it possible that it was reported on HiDPI screen macOS?

Reply:
It was reported in macmini (probably connect to hidpi screen) and iMac (I guess having hidpi screen).
Also, I have seen mach5 job failing on windows too.
I have posted failing mach5 jobs in JBS.

Reply:
Then please double-check that the test will pass after the fix on HiDPI screen. Not sure that -Dsun.java2d.uiScale=1 affects such usecase.

Reply:
Since mach5 systems are mostly macmini, where it initially failed, and the resultant test is passing on them, I would hope they will pass on hidpi screen they are connected to.

Reply:
The mach5 macOS test systems currently have ZERO hidpi monitors. I think they are all 1920x1080 non-retina.
They only ever had at most one and that was because it was accidentally connected that way after a lab move.